docs/search.md

```diff
@@ -0,0 +1,24 @@
+# Search
+
+## Basic search
+
+The user enters a search keyword and the application parses through all the searchable fields and returns a list of search results that contain the keyword.
```

Comment:
"You can enter...", because this is aimed at editors/users, so it's better to address them directly.
Also, maybe replace "parse" with a less specialised term? Remember the audience in this case.